com.sigmyne.crush.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "com.sigmyne.crush",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk" ],
+    "command": "crush",
+    "finish-args": [
+   	"--share=network",
```

Comment:
Whitespace problems throughout the file.

Reply:
Thanks. I replaced leading white spaces with tabs. Hopefully that's what you meant...